modules/apps/product-navigation/product-navigation-user-personal-bar-web/src/main/resources/META-INF/resources/js/index.js

```diff
@@ -92,6 +92,14 @@ export function signInButtonPropsTransformer({
 							document.querySelector('.loading-animation')
 						) {
 							updateModalContent(modalContentHTML);
+							document.getElementsByClassName(
+								'btn disabled btn-primary'
+							)[0].disabled = false;
+							document
+								.getElementsByClassName(
+									'btn disabled btn-primary'
+								)[0]
+								.classList.remove('disabled');
```

Comment:
same here, please store the 
```
document.getElementsByClassName(
		'btn disabled btn-primary'
	)[0]
``` 
in a variable

Reply:
Hi @smottal ,
Thanks for the review :)
I've created the variable and a little playwright test. Let me know if further changes are needed